plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1808,7 +1867,7 @@ bool producer_plugin_impl::process_unapplied_trxs( const fc::time_point& deadlin
             }
 
             auto prev_billed_cpu_time_us = trx->billed_cpu_time_us;
-            if( !_subjective_billing.is_disabled() && prev_billed_cpu_time_us > 0 && !rl.is_unlimited_cpu( first_auth )) {
```

Comment:
This should not have been removed. Also needs to be added to develop branch PR.

Reply:
Fixed.